transformations/replace_financial_amounts/entity_financial_amount.py

```diff
@@ -0,0 +1,276 @@
+from typing import List, Dict
+import random
+import re
+import os
+
+
+__location__ = os.path.realpath(
+    os.path.join(os.getcwd(), os.path.dirname(__file__))
+)
+
+
+def load_currency_abbreviation_file(path: str) -> List[str]:
+    """
+    Returns a list of currency abbreviations present as one value per line in a file.
+    """
+    with open(
+        os.path.join(__location__, path), "r", encoding="utf-8"
+    ) as _file:
+        file_content = _file.read()
+        result = file_content.split("\n")
+
+    if result[-1] == "":
+        return result[:-1]
+
+    return result
+
+
+CURRENCY_SYMBOLS = ["$", "£", "₤", "¥", "฿", "€"]
+
+CURRENCY_ALL = [
+    "dollar",
+    "dollars",
+    "pound",
+    "pounds",
+    "yen",
+    "yens",
+    "yuan",
+    "bitcoin",
+    "bitcoins",
+    "euro",
+    "euros",
+] + CURRENCY_SYMBOLS
+
+CURRENCY_ABBREVIATIONS = load_currency_abbreviation_file("currency_abbr.txt")
+
+# The exchange rate used bellow are those used against 1 USD as of August 2021.
+CURRENCIES = {
+    "dollar": {
+        "name": "dollar",
+        "symbol": ["$", "dollar", "dollars", "USD", "US dollars", "Dollars"],
+        "rate": 1.0,
```

Comment:
Should the rates be easily editable through a config file or a config function (at initialization for example) to be modified by the client/caller?